pallets/api/src/fungibles/tests.rs

```diff
@@ -341,7 +341,7 @@ fn start_destroy_works() {
 		// Check that the token is not live after starting the destroy process.
 		assert_noop!(
 			Assets::mint(signed(ALICE), token, ALICE, 10 * UNIT),
-			AssetsError::AssetNotLive
+			TokenError::UnknownAsset
```

Comment:
Test was failing due to the following https://github.com/paritytech/polkadot-sdk/pull/3286

Concerning to me is that this would have broken a smart contract. Presumably contracts won't often match on a specific error but if so, the contract could behave unexpected.

Reply:
This error only changed for the `mint` function. 

The reason why it is concerning to me is that I don't have the feeling parity is seeing this as a breaking change, meaning that they'll make such changes more often, meaning that we have to create more versions.

Reply:
We have decided to tackle this later. The only solution would be to add a conversion for the specific error and query whether the asset is in destroy mode. With only adding the conversion it would change the error in all cases which it shouldn't. A discussion has been started to see how to move forward here: https://forum.polkadot.network/t/discussion-treating-changes-in-dispatchable-function-errors-as-breaking-changes/11222/4